src/arduinofw/ArduinoI2C/src/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include <Arduino.h>
+#include <Wire.h>
+
+void onRecv(int ln) {
+
+	while(Wire.available()) {
+		if (Wire.read() == 0x0) {
+			PORTB &= ~(1 << 5);
+			PORTB |= (Wire.read() & 0x1) << 5;
```

Comment:
We could have use PORTD and DDRD right? Or was a specific choice the use of PORTB and DDRB?

Reply:
I am using them because it's where the onboard led is connected (pin 13). However the pin 13 is also available externally so a relay can be directly connected there and will work without changing the firmware, so you have both th led and the relay working in the same pin

Reply:
Ok clear, thanks!